modules/understand-etcd-performance.adoc

```diff
@@ -0,0 +1,124 @@
+// Module included in the following assemblies:
+// etcd/etcd-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="understand-etcd-performance_{context}"]
+= Understanding etcd performance
+
+As a consistent distributed key-value store operating as a cluster of replicated nodes, etcd follows the Raft algorithm by electing one node as the leader and the others as followers. The leader maintains the system's current state and ensures that the followers are up-to-date.
+
+The leader node is responsible for log replication. It handles incoming write transactions from the client and writes a Raft log entry that it then broadcasts to the followers. 
+
+When an etcd client such as `kube-apiserver` connects to an etcd member that is requesting an action that requires a quorum, such as writing a value, if the etcd member is a follower, it returns a message indicating that the transaction should go to the leader. 
```

Comment:
```suggestion
When an etcd client such as `kube-apiserver` connects to an etcd member that is requesting an action that requires a quorum, such as writing a value, if the etcd member is a follower, it returns a message indicating that the transaction needs to go to the leader. 
```
Avoiding "should", per ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#should